packages/grafana-ui/src/components/Monaco/utils.ts

```diff
@@ -0,0 +1,17 @@
+import { VariableSuggestion } from '@grafana/data';
+import { CodeEditorSuggestionItem, CodeEditorSuggestionItemKind } from './types';
+
+/**
+ * @alpha
+ */
+export function variableSuggestionToCodeEditorSuggestion(sug: VariableSuggestion): CodeEditorSuggestionItem {
```

Comment:
Not a huge fan of this mapping. I mean we could probably try to unify the VariablesSuggestion and CodeEditorSuggestionItem to have one interface. I'll try to do that.